frontend/src/components/admin.js

```diff
@@ -81,11 +81,11 @@ function GuestsCard() {
     if (isLoading) {
       axios.post(process.env.REACT_APP_API_ENDPOINT+'/api/create_guests/', { jwt: jwt['jwt'] }).then((res) => {
         console.log(res.data);
-        setLoading(false);
-        window.location = "/admin"
+	setLoading(false);
+	setPhrase("{" + res.data.results.join("\n") + "}");
```

Comment:
Think wants `setPhrase({res.data.results})` 

Then `{phrase.map((item) => {<p>{item}</p>})}

Assuming the results is an Array

Reply:
yea `results` is an array..... played with some variants of what you suggested and no joy. will have to go deeper into the depths of react, i guess.

Reply:
https://www.geeksforgeeks.org/how-to-map-data-into-components-using-reactjs/